src/view/drag-handle/sensor/create-touch-sensor.js

```diff
@@ -135,10 +135,52 @@ export default ({
     kill(callbacks.onCancel);
   };
 
+  // Safari 11.3 hack
+  // Safari does not allow event.preventDefault() in dynamically added handlers
+  // So we add an always listening event handler to get around this :(
+  // webkit bug: https://bugs.webkit.org/show_bug.cgi?id=184250
+  (() => {
+    // Do nothing when server side rendering
+    if (typeof window === 'undefined') {
+      return;
+    }
+
+    const isUsingSafari11: RegExp = /AppleWebKit.*Version\/11/g;
+
+    // Not using Safari 11
+    if (!isUsingSafari11.test(window.navigator.userAgent)) {
+      return;
+    }
+
+    // Using Safari 11 with no touch support - no point adding the touch listeners
+    if (!('ontouchstart' in window)) {
+      return;
+    }
+
+    // Adding a persistent event handler
+    window.addEventListener('touchmove', (event: TouchEvent) => {
```

Comment:
Is there any part of the lifecycle where we would want to remove the event listener? E.g. on unmount?